webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingFormDrawer.vue

```diff
@@ -214,14 +223,20 @@ export default {
       }
     },
     saveSpace(event) {
-      if (event) {
-        event.preventDefault();
-        event.stopPropagation();
+      const regex = /^[A-Za-z0-9& ]+$/;
```

Comment:
Please use the same pattern as in org.exoplatform.social.core.space.SpaceUtils#isValidSpaceName